sdl-core/pom.xml

```diff
@@ -340,6 +340,25 @@
 			<version>${json4s.version}</version>
 		</dependency>
 
+		<!-- Dependency for StatusInfoRestApi-->
+		<dependency>
+			<groupId>org.eclipse.jetty</groupId>
+			<artifactId>jetty-servlet</artifactId>
+			<version>9.4.43.v20210629</version>
+		</dependency>
+		<!-- Dependency for StatusInfoRestApi-->
+		<dependency>
+			<groupId>org.eclipse.jetty</groupId>
+			<artifactId>jetty-http</artifactId>
+			<version>9.4.43.v20210629</version>
+		</dependency>
+		<!-- Needed to serialize Requests for StatusInfoRestApi-->
+		<dependency>
```

Comment:
Should we use ${jersey.version} as version for this library?

Reply:
OK, I reference now ${jetty.version} from pom.xml